modules/lambda-cloudwatch/main.tf

```diff
@@ -1,3 +1,7 @@
+terraform {
+  required_version = ">= 0.12.0"
```

Comment:
Since we expect this to be over a certain version for terraform and we know the earlier versions are getting deprecated provider wise we might always want to add a constraint for the aws provider as well found here https://www.terraform.io/docs/configuration/terraform.html#specifying-required-provider-versions

```
terraform {
  required_providers {
    aws = ">= 2.7.0"
  }
}
```

It is a tad more explicit but it may be worth figuring out what our minimum version is based on some of the VPC changes that were implemented.

Reply:
Are we sure ">= 2.7.0" is the correct requirement for everything? If so I'm good with adding it.

Reply:
Oh that was just an example sorry I don't know what minimum version is off hand.